DemoApp/StreamChat/Components/DemoChatChannelListVC.swift

```diff
@@ -159,7 +159,8 @@ final class DemoChatChannelListVC: ChatChannelListVC, EventsControllerDelegate {
     override func controller(_ controller: DataController, didChangeState state: DataController.State) {
         super.controller(controller, didChangeState: state)
 
-        if highlightSelectedChannel && (state == .remoteDataFetched || state == .localDataFetched) && selectedChannel == nil {
+        // TODO: This is actually a breaking change
+        if highlightSelectedChannel && (state.isRemoteDataFetched || state == .localDataFetched) && selectedChannel == nil {
```

Comment:
This is the only breaking change, the customers can't do `state == .remoteDataFetched` anymore. But this is easily fixed by changing to `state.isRemoteDataFetched`. I wouldn't mind this minor breaking change TBH